### PR TITLE
initial upgrade of the legacy note

### PR DESCRIPTION
**Pull request by Bonnarel**
Upgrade  citations of most recent VO specifications and adding a section on usage of response FIELDS

### CODE REVIEW

File: DataLinkImp.tex [hunk 1]
```diff
@@ -60,20 +60,19 @@ \section*{Conformance-related definitions}
...
+The \blinks\ web service capability is a web service capability for drilling
 down from a discovered data item such as  a dataset identifier, a source in a
 catalog or any other data item. In the first case (typically an IVOA publisher
 dataset identifier) it allows to find details about the data files that can be
 downloaded, alternate representations of the data that are available, and
 services that can act upon the data (usually without having to download
 the entire dataset). In the latter cases it allows to associate metadata, images,
 cubes, spectra, timeseries or ancillary data to a source in the sky or other
-kind of time. The expected usage is for DAL (Data Access Layer)
+kind of iteim. The expected usage is for DAL (Data Access Layer)
```
Comment (gmantele) [inline DataLinkImp.tex]: Are you sure of the word `iteim`?

**Review by gmantele — COMMENTED** [review]
By the way, are you aware you created this PullRequest as a **Draft**?
It means it cannot be merged event with approvals.

**Comment by Bonnarel** [comment]
Le 23/01/2024 à 16:18, Grégory Mantelet a écrit :
>
> ***@***.**** commented on this pull request.
>
> By the way, are you aware you created this PullRequest as a *Draft*?
>
no. I go back to this and change the typo "iteim" to "item" (and not time)

Thanks Gregory

**Comment by Bonnarel** [comment]
It was a draft PR

**Pull request closed without merging**